csvcubed/csvcubed/cli/entrypoint.py

```diff
@@ -0,0 +1,83 @@
+"""
+CLI
+---
+The *Command Line Interface* for :mod:`~csvcubed.csvcubedcli.infojson2csvqb`.
+"""
+from email.policy import default
+import logging
+import click
+from pathlib import Path
+import colorama
+from csvcubed.utils.log import start_logging
+
+from .build import build
+
+
+@click.group(context_settings=dict(help_option_names=["-h", "--help"]))
+def entry_point():
+    """
+    csvcubed - a tool to generate qb-flavoured CSV-W cubes from COGS-style info.json files.
+    """
+    colorama.init(autoreset=True, wrap=True)
+
+
+@entry_point.command("build")
+@click.option(
+    "--config",
+    "-c",
+    help="Location of the info.json file containing the QB column mapping definitions.",
```

Comment:
`info.json` is no more. I'd also get rid of references to the `qb columns` and just call them `column definitions` tbh.